packages/fxa-settings/src/components/App/HeaderContinueTo/en.ftl

```diff
@@ -0,0 +1,10 @@
+## HeaderContinueTo component
+
+# This message directly follows a task header (e.g., 'Reset password')
+# If more appropriate in a locale, this can stand alone as "Continue to { $serviceName }"
+header-continue-to-default-subheading = to continue to account settings
```

Comment:
GetText didn't allow us to do the right thing, which is exposing the complete sentence instead of fragments. This approach of patching together parts is less than ideal for localization.

Could we expose the full text here with markup as needed? The downside is that there's no guarantee that the task description will be at the start of the sentence.

```
<span>Reset password</span> to continue to account settings
```

Or maybe we need a different approach even for English, with two separate strings.

```
Reset password
You need to reset your password to continue to account settings
```

In any case, the comment shouldn't mention `$serviceName` for this string.

Reply:
All very good points - I don't think it makes a difference if the task description is at the beginning or the end of the sentence, so I'll combine these into one heading element/string.